proxysql-admin

```diff
@@ -3203,6 +3252,9 @@ function main() {
   elif [[ $REPORT_STATUS -eq 1 ]]; then
 
     report_status $WRITER_HOSTGROUP_ID
+  elif [[ $UPDATE_MYSQL_VERSION -eq 1 ]]; then
```

Comment:
Need to remove this elif (it's duplicated below)

Reply:
Done